### DISCUSSION TITLE
.NET package for tidal's public API

### DISCUSSION THREAD

**Discussion by monke8555** [tidal-music/discussions#79] [Ideas & Feature requests]
I wanted to write a simple program in C# that would sort my playlists with the tidal api, but realised there was no decent library for C#, and started to write it manually for the endpoints I needed. But I got carried away 😅 and wrote a full library for all the non-personalised and user apis, and I was wondering if I can submit them to the official tidal org for it to become an accepted repo in the org.

**Comment by stefanmoro** [comment]
Hi @monke8555, thanks for reaching out, and great to hear you are building on our API!

We currently do not have the possibility to add libs/code written outside TIDAL to our repo, but we might open up for that in the future. In the meantime, feel free to post a link to your library in this discussion forum for others to check it out!

Thank you!

**Comment by 6a6f6a6f** [comment]
Hello @monke8555 I'm **very** interested in writing the library with you. I have a couple wrappers with C# AOT for macOS and Windows Tidal apps, but nothing to the REST API yet. Please let me know if we can get started on that and create an unofficial client library.
